### DISCUSSION TITLE
I'm a verified student but can't use the GitHub Student Developer Pack. What do I do now?

### DISCUSSION THREAD

**Discussion by developer-rashed** [community/community#124980] [GitHub Education]
### Hi everyone,

I'm a verified student but can't use the GitHub Student Developer Pack. What do I do now?

![App git](https://github.com/community/community/assets/70908832/b6f58d3e-6eb9-48d2-9ff8-fc6507e3c18d)
![git](https://github.com/community/community/assets/70908832/87ee4e5c-7a6b-475d-9ad8-cb3c04f47ff8)

**Comment by davevad93** [comment]
Hi @developer-rashed ,

That's odd if you are a verified student with access to `pro` features you should definitely have access to the `student developer pack`.
Have you tried connecting your GitHub account to any of these offers? If you can access them you definitely have access to your student benefits.

![edu](https://github.com/community/community/assets/94308234/00c983fa-8da6-4613-ba53-f038e5f57325)
